process-phoenix/src/main/java/com/jakewharton/processphoenix/ProcessPhoenix.java

```diff
@@ -59,7 +59,7 @@ public static void triggerRebirth(Context context, Intent nextIntent) {
     intent.addFlags(FLAG_ACTIVITY_NEW_TASK); // In case we are called with non-Activity context.
     intent.putExtra(KEY_RESTART_INTENT, nextIntent);
     context.startActivity(intent);
-
+    ((Activity) context).finish();
```

Comment:
My bad. :\
So in triggerRebirth method, 
Should we trigger an exception if user passes non-activity context? 
Or we simply ignore it using instanceof check? 
Or we change the signature to pass an activity?
What do you suggest?